### ISSUE TITLE
[osu!mania]: Counter for Mania Millions

### ISSUE THREAD

**Issue by WindowsMeosu** [respektive/osu-tracker#15]
Self-explanatory: A stat that counts how many total 1,000,000 scores achieved.

**Comment by respektive** [comment]
I don't think this would be possible to add to the tracker. osu! doesn't show the total amount of millions anywhere on the site and thus it is also not in the API response. The only way to track this is to check every score a user has and see how many of those are 1 million, which would mean to check every ranked and loved beatmap. Way too many requests to properly track as well, you could probably only track this about once a day at best.

There are sites that already track this like https://osustats.ppy.sh/maniaMillions or https://osudaily.net/ranking.php?t=mill that update around once a day from what I can tell. I might be able to use the osudaily API in the tracker to show the Count in the tracker like that, but I'm not sure how useful that actually is, when it's not updating in real time like the other stats. :thinking:

**Comment by WindowsMeosu** [comment]
> The only way to track this is to check every score a user has and see how many of those are 1 million

 not necessary every score submitted, only scores set in the top1000

> I might be able to use the osudaily API in the tracker to show the Count in the tracker like that, but I'm not sure how useful that actually is, when it's not updating in real time like the other stats.

As you implied, osu!daily is certainly not going to be useful for tracking, especially when you consider that the database is rarely updated, or more accurately, abandoned.
![image](https://github.com/respektive/osu-tracker/assets/104236864/41648a97-9c34-415b-8087-0afd90ffd441)
![image](https://github.com/respektive/osu-tracker/assets/104236864/3d8e926e-789e-4907-afbe-9e23edef9654) (I have 39 top1k 1mils total counting from [osu!stats](https://osustats.ppy.sh/u/Windows%20Me///3/2//0-5/-/1-1000/)).

Despite everything, I'm pretty sure this isn't impossible to implement, and I'm 100% sure there's a way without putting stress on the server.
